.github/workflows/release_chart.yaml

```diff
@@ -31,8 +32,21 @@ jobs:
         run: |
           helm repo add bitnami https://charts.bitnami.com/bitnami
 
+      - name: Set environment variables for chart-releaser
+        run: |
+          if [ "${{ github.event.workflow_run.head_branch }}" == "dev" ]; then
+            echo "MARK_AS_LATEST=false" >> $GITHUB_ENV
+            echo "CR_RELEASE_NAME_TEMPLATE={{ .Name }}-{{ .Version }}-dev" >> $GITHUB_ENV
+          else
+            echo "MARK_AS_LATEST=true" >> $GITHUB_ENV
+            echo "CR_RELEASE_NAME_TEMPLATE={{ .Name }}-{{ .Version }}" >> $GITHUB_ENV
+          fi
+
       - name: Run chart-releaser
         uses: helm/chart-releaser-action@v1.3.0
+        with:
+          mark_as_latest: ${{ env.MARK_AS_LATEST }}
```

Comment:
what happens if you just do:

```
mark_as_latest: "${{ github.event.workflow_run.head_branch }}" == "main"
```

Reply:
that should work, is there any way to test these before they're merged in?

Reply:
I found and ran `act` with your suggestion and got an error: ```Error: workflow is not valid. 'release_chart.yaml': yaml: line 47: did not find expected key```!

Reply:
here's the `act` output for the step I added:

[Release Charts/release]   ✅  Success - Main Set environment variables for chart-releaser
[Release Charts/release]   ⚙  ::set-env:: MARK_AS_LATEST=true
[Release Charts/release]   ⚙  ::set-env:: CR_RELEASE_NAME_TEMPLATE={{ .Name }}-{{ .Version }}

Reply:
what if we the syntax from my suggestion to something like:

```
"${{ github.event.workflow_run.head_branch  == 'main' }}"
```

using `MARK_AS_LATEST` seems more verbose than required, at least to my eyes!

Reply:
Good find on `act`! I've seen it floating around but never tried it.  historically testing GitHub actions has been the actual worst so good to know that there's an option that seems to work!